test/e2e/default_bridged_network_test.go

```diff
@@ -43,43 +44,43 @@ var _ = Describe("NodeNetworkConfigurationPolicy default bridged network", func(
   - name: brext
     type: linux-bridge
     state: absent
-`)
+`, *primaryNic))
 	Context("when there is a default interface with dynamic address", func() {
 		addressByNode := map[string]string{}
 		BeforeEach(func() {
-			By("Check eth0 is the default route interface and has dynamic address")
+			By(fmt.Sprintf("Check %s is the default route interface and has dynamic address", *primaryNic))
```

Comment:
Man, someone should create `Byf(msg, args...)` ...

Reply:
https://github.com/nmstate/kubernetes-nmstate/issues/212

Reply:
Agree, thanks for the issue.